src/transaction.js

```diff
@@ -100,6 +100,17 @@ function mkScriptHashReplayScript (
   )
 }
 
+function mkReplayScript(pubKeyHash: string): string {
```

Comment:
What type of replay script is this?

Reply:
is a pay to public key hash "standard" (without the replay protection)
should be correct  to use it in certificates because the vout of certificates don't have the replay protection.
I'd change the method name to avoid misreadings